src/algebra/ordered_group.lean

```diff
@@ -443,138 +494,293 @@ def function.injective.ordered_comm_group {β : Type*}
+section right
+variables [covariant_class α α (function.swap (*)) (<)] {a b c d : α}
+
+@[simp, to_additive]
+lemma div_lt_div_iff_right (c : α) : a / c < b / c ↔ a < b :=
+by simpa only [div_eq_mul_inv] using mul_lt_mul_iff_right _
+
+@[to_additive sub_lt_sub_right]
+lemma div_lt_div_right' (h : a < b) (c : α) : a / c < b / c :=
+(div_lt_div_iff_right c).2 h
+
+@[simp, to_additive sub_pos]
+lemma one_lt_div' : 1 < a / b ↔ b < a :=
+by rw [← mul_lt_mul_iff_right b, one_mul, div_eq_mul_inv, inv_mul_cancel_right]
+
+alias sub_pos ↔ lt_of_sub_pos sub_pos_of_lt
 
-lemma lt_sub_iff_add_lt : a < c - b ↔ a + b < c :=
-by rw [lt_sub_iff_add_lt', add_comm]
+@[simp, to_additive sub_neg]
+lemma div_lt_one' : a / b < 1 ↔ a < b :=
+by rw [← mul_lt_mul_iff_right b, one_mul, div_eq_mul_inv, inv_mul_cancel_right]
+
+alias sub_neg ↔ lt_of_sub_neg sub_neg_of_lt
+
+alias sub_neg ← sub_lt_zero
+
+@[to_additive]
+lemma lt_div_iff_mul_lt : a < c / b ↔ a * b < c :=
+by rw [← mul_lt_mul_iff_right b, div_eq_mul_inv, inv_mul_cancel_right]
 
 alias lt_sub_iff_add_lt ↔ add_lt_of_lt_sub_right lt_sub_right_of_add_lt
 
-lemma sub_lt_iff_lt_add' : a - b < c ↔ a < b + c :=
-by rw [sub_eq_add_neg, add_comm, neg_add_lt_iff_lt_add]
+@[to_additive]
+lemma div_lt_iff_lt_mul : a / c < b ↔ a < b * c :=
+by rw [← mul_lt_mul_iff_right c, div_eq_mul_inv, inv_mul_cancel_right]
+
+alias sub_lt_iff_lt_add ↔ lt_add_of_sub_right_lt sub_right_lt_of_lt_add
+
+end right
+
+section left
+variables [covariant_class α α (*) (<)] [covariant_class α α (function.swap (*)) (<)] {a b c : α}
+
+@[simp, to_additive]
+lemma div_lt_div_iff_left (a : α) : a / b < a / c ↔ c < b :=
+by rw [div_eq_mul_inv, div_eq_mul_inv, ← mul_lt_mul_iff_left a⁻¹, inv_mul_cancel_left,
+    inv_mul_cancel_left, inv_lt_inv_iff]
+
+@[to_additive sub_lt_sub_left]
+lemma div_lt_div_left' (h : a < b) (c : α) : c / b < c / a :=
+(div_lt_div_iff_left c).2 h
+
+end left
+
+end group
+
+section comm_group
+variables [comm_group α]
+
+section has_lt
+variables [has_lt α] [covariant_class α α (*) (<)] {a b c d : α}
+
+@[to_additive sub_lt_sub_iff]
+lemma div_lt_div_iff' : a / b < c / d ↔ a * d < c * b :=
+by simpa only [div_eq_mul_inv] using mul_inv_lt_mul_inv_iff'
+
+@[to_additive]
+lemma lt_div_iff_mul_lt' : b < c / a ↔ a * b < c :=
+by rw [lt_div_iff_mul_lt, mul_comm]
+
+alias lt_sub_iff_add_lt' ↔ add_lt_of_lt_sub_left lt_sub_left_of_add_lt
+
+@[to_additive]
+lemma div_lt_iff_lt_mul' : a / b < c ↔ a < b * c :=
+by rw [div_lt_iff_lt_mul, mul_comm]
 
 alias sub_lt_iff_lt_add' ↔ lt_add_of_sub_left_lt sub_left_lt_of_lt_add
 
-lemma sub_lt_iff_lt_add : a - c < b ↔ a < b + c :=
-by rw [sub_lt_iff_lt_add', add_comm]
+@[simp, to_additive]
+lemma inv_lt_div_iff_lt_mul : b⁻¹ < a / c ↔ c < a * b :=
+lt_div_iff_mul_lt.trans inv_mul_lt_iff_lt_mul'
 
-alias sub_lt_iff_lt_add ↔ lt_add_of_sub_right_lt sub_right_lt_of_lt_add
+@[to_additive]
+lemma inv_lt_div_iff_lt_mul' : a⁻¹ < b / c ↔ c < a * b :=
+by rw [inv_lt_div_iff_lt_mul, mul_comm]
```

Comment:
I'm quite sure this one doesn't need commutativity. I would be much happier if this lemma is the `simp`, instead of `inv_lt_div_iff_lt_mul` (and if this becomes the unprimed version)

Reply:
Proved with no commutativity, renamed and swapped `simp` tag.